README.md

```diff
@@ -15,6 +15,7 @@ Material Components for the web is the successor to [Material Design Lite](https
   - [Preact Material Components](https://github.com/prateekbh/preact-material-components)
   - [RMWC: React Material Web Components](https://github.com/jamesmfriedman/rmwc)
   - [Blox Material](https://blox.src.zone/material): Angular Integration Library.
+  - [Material Components Vue](https://github.com/matsp/material-components-vue)
```

Comment:
Can you add a note about how this uses the Vanilla components approach. This will help clarify the main differences between this Vue library, and stasson's version